[caffe2] Wrap constexpr with preprocessor statements

danzimm opened the pull request:
Summary: When the preprocessor check we leave an unused constexpr around, so when `-Wunused-const-variable` is enabled we get an error. Let's inline these values since they're not used anywhere else in order to avoid this.

Test Plan: CI

Differential Revision: D60723823

houseroad reviewed (approved):
Try rerun the tests, or rebase?

danzimm committed:
[caffe2] Wrap constexpr with preprocessor statements (#132582)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/132582

When the preprocessor check we leave an unused constexpr around, so when `-Wunused-const-variable` is enabled we get an error. Let's inline these values since they're not used anywhere else in order to avoid this.

Test Plan: CI

Reviewed By: houseroad

Differential Revision: D60723823
